resourcepolicies.md

```diff
@@ -10,34 +10,298 @@ As we don't have yet an use case to iterate over all the existent resource polic
+* 200 OK - if the operation succeed
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions. Only system administrators and users with ADMIN right on the target resource can use the endpoint
+* 422 UNPROCESSABLE ENTITY - if the uuid parameter is missing or invalid
+
+#### eperson
+**/api/authz/resourcepolicies/search/eperson?uuid=<:uuid>[&resource=<:resource>]**
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* uuid: mandatory, the uuid of the eperson that benefit of the policy
+* resource: optional, limit the returned policies to the ones related to the specified resource
+
+It returns the list of matching resource policies
+
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions. Only system administrators, the user specified in the uuid parameter or user with ADMIN permission over such user can use the endpoint
+* 422 UNPROCESSABLE ENTITY - if the uuid parameter is missing or invalid
+
+#### group
+**/api/authz/resourcepolicies/search/group?uuid=<:uuid>[&resource=<:resource>]**
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* uuid: mandatory, the uuid of the eperson that benefit of the policy
+* resource: optional, limit the returned policies to the ones related to the specified resource
+
+It returns the list of matching resource policies
+
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions. Only system administrators, users member of the group specified in the uuid parameter or user with ADMIN permission over it can use the endpoint
+* 422 UNPROCESSABLE ENTITY - if the uuid parameter is missing or invalid
+
+#### authorizeAnyOf
+**/api/authz/resourcepolicies/search/authorizeAnyOf?resource=<:resource>&action=<:action>&action=<:actionN>[&useInheritance=<:boolean>]**
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination). MUST be size = 0, the endpoint *currently* never return a list of resourcepolicies, in future it could be possible to return the list of resourcepolicy that collectively grant the current user the requested actions
+* resource: mandatory, the uuid of the resource that need to be checked for permission
+* action: multiple, at least one required. The actions that need to be verified
+* useInheritance: optional, assume false as default. True mean that policies on parents objects are considered, false will focus only on policy attached directly to the resource
+
+It would returns the list of resource policies that grant to the user the requested actions of the resource. When size = 0 the pagination totalElements attribute in the response can be used to discriminate if all the actions are allowed or not.
```

Comment:
How is the *user* specified?  I'd expect to see a required `eperson` param or similar.

Reply:
yes, @benbosman  is right I'm making this explicit